ee/clickhouse/queries/test/__snapshots__/test_retention.ambr

```diff
@@ -364,15 +376,8 @@
           NULL as selected_interval,
           returning_event_query as
        (SELECT toStartOfWeek(toTimeZone(toDateTime(e.timestamp, 'UTC'), 'UTC')) AS event_date,
-               pdi.person_id as target
+               e.person_id as target
         FROM events e
-        INNER JOIN
```

Comment:
should this be changing?

Reply:
Actually, this particular one yes. Somehow this was wrong on master? Note how it now correctly uses PoE v1 and it is a `person_on_events` test